src/rpdk/core/contract/resource_client.py

```diff
@@ -137,7 +137,6 @@ def __init__(
         docker_image=None,
         executable_entrypoint=None,
     ):  # pylint: disable=too-many-arguments
-        self._schema = schema
```

Comment:
Why was this removed?

Reply:
because this line doesn't serve any purpose. See line 168